### ISSUE TITLE
In Libertinus Serif, J looks different in bold italic face.

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [khaledhosny/libertinus#98]
![1](https://i.imgur.com/z88ZiD9.png)
Also see: https://sourceforge.net/p/linuxlibertine/bugs/326/

**Comment by mgavioli** [comment]
To my taste, it is the Italic and the Semi-bold italic 'J' which look out of context, with their swash appearance, not shared by any other Latin upper case glyph.

My suggestion would be to 'straighten' the first two and leave the third as it is. Possibly, the current Italic and Semi-bold italic form could be moved to a stylistic set (or feature) of some sort (just not to loose them).

**Comment by khaledhosny** [comment]
I’d rather have the swashy one in all the serif italic fonts by default, and provide the other form as stylistic alternate.

**Comment by KrasnayaPloshchad** [comment]
@khaledhosny I agree with you. 🤝 And I suggest using `ssXX` feature would be a nice choice.

**Comment by mgavioli** [comment]
Current status:

![issue_098_screenshot_01](https://user-images.githubusercontent.com/1420743/37180282-be454a18-2328-11e8-8305-ed863facf030.png)

- the swashy shape is the same in Sans and in Serif;
- In all variants, except Serif Italic and Serif Semi-bold italic, the 'straight' shape is the default;
- In Sans Regular and Italic and in Serif Regular and Italic, the 'other' shape (3 of of 4, the swashy one) is in `J.alt`
- Sans Bold, both Serif Semi-bold and both Serif Bold lack the 'other' shape.

Four glyphs need to be drawn from scratch:
- 'swashy' bold (for Sans and for Serif)
- 'swashy' semi-bold
- 'straight' semi-bold italic
- 'swashy' bold italic

From the over-all picture, I am still convinced that the 'straight' should be the default everywhere and the 'swashy' can go in `salt`, but this is relatively easy to fix even as a second step.

**Comment by khaledhosny** [comment]
* The swash form in sans make no sense (and the existing glyphs are just copies of serif ones), so we should just drop them.
* Serif regular should continue to have the straight form as default.
* Swash form should be default in serif italic, and I'm OK with serif roman not having an alternate form (but it is OK to have one).

In general I don’t think we should change the defaults too much and try to maintain the existing feel of the fonts as much as possible, unless we are fixing real bugs.

**Comment by waldyrious** [comment]
IMHO, I agree with @mgavioli that it would be better to have the "swashy" shape always be the alt glyph. I don't think it would be deviating too much from the original font, but rather resolving a styling inconsistency already present in it. I see this this as a reasonable *specific* deviation from the *general* principle "we should change the defaults too much", which which I agree as well.

**Comment by Ransom77** [comment]
Once I took a closer look at this discussion thread, it seemed to me that there were several sets from Libertinus Serif. Then when I looked at the OT features, I discovered the Stylistic sets and tried them out: And lo and behold, if you use Stylistic Set 2 for the merworthy “J” in the italic version, you get the “real” J.

Now I entered all capital letters once, sometimes with Stylistic Set 1, sometimes with Stylistic Set 2.
![Beispiel](https://user-images.githubusercontent.com/66835201/207911420-56d9dd27-2723-4c7f-817e-76d7c8271cfa.png)
It seems to me that the “J” has been swapped in the two sets. Wouldn’t it be easiest (and make the most sense) to swap the two “J”s in the two sets?

Anyway. I’m glad that I can get rid of this strange (and in my eyes totally out of place) “J” in the italic version with “Libertinus Serif:ss02” resp. get the “right” one.
